COVIDWatch iOS/UIComponents/Button.swift

```diff
@@ -74,6 +74,18 @@ class Button: UIView {
         }
     }
 
+    func drawBetween(parentVC: UIViewController, top: CGFloat, bottom: CGFloat, centerX: CGFloat) {
+//        Draw the button so that it and its subtext taken together are
+//        centered between top and bottom
+        self.draw(parentVC: parentVC, centerX: centerX, centerY: (top+bottom)/2)
+        if self.subtext != nil {
+            let adjustment = (self.subtext!.frame.maxY - self.frame.maxY) / 2
+            self.center.y -= adjustment
+            self.text.center.y -= adjustment
+            self.subtext!.center.y -= adjustment
```

Comment:
Can this be an if let instead of force unwrapping please?

Reply:
I don't understand what that suggestion means (Swift noob)

Reply:
Never mind I read it wrong